src/style.css

```
@@ -1 +1,101 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+}
+
+.icon {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+  cursor: pointer;
+}
+
+.cry {
+  display: block;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.menu {
+  position: relative;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding-left: 0;
+}
+
+.list {
+  padding-top: 19px;
+  margin-left: 20px;
+  padding-bottom: 19px;
+}
+
+.list:first-child {
+  margin-left: 0;
+}
+
+.link {
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  line-height: 14px;
+  color: #000;
+  text-transform: uppercase;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.link::after {
+  display: block;
+  content: "";
+  position: absolute;
+  left: 0;
+  top: 33px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  height: 4px;
+  width: 0;
+}
+
+.link:hover::after {
+  width: 100%;
+}
+
+.is-active {
+  color: #00acdc;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  top: 33px;
```

```suggestion
  bottom: 0;
```
+ position relative for link